game-core/src/main/java/games/strategy/debug/ErrorMessage.java

```diff
@@ -67,31 +71,28 @@ public void windowClosing(final WindowEvent e) {
                 .toolTip("Shows the error console window with full error details.")
                 .actionListener(() -> {
                   hide();
-                  ErrorConsole.showConsole();
+                  ClientSetting.SHOW_CONSOLE.saveAndFlush("true");
```

Comment:
When we open the console window explicitly, TripleA will now remember that decision. There is similar tear down code to remember for it to be closed when we close the console.